lib/msal-browser/docs/access-token-proof-of-possession.md

```diff
@@ -151,6 +151,10 @@ The Proof-of-Possession authentication scheme relies on an asymmetric cryptograp
 
 In the event of refreshing a bound access token, MSAL will delete the cryptographic keypair that was generated when requesting the expired bound access token, generate a new cryptographic keypair for the new access token, and store the new keypair in the keystore.
 
+## Advanced feature: User generated cryptographic keypair
+
+MSAL does not sign the POP tokens if the application chooses to generate its own random cryptographic keypair and returns the unsigned `access_token` generated by the Service to the application. This is fairly advanced feature and we recommend not to use it unless the developer understands POP and finds a use case on their end to manage the `reqCnf` generation themselves. `reqCnf` can be provided as an optional parameter on the `request` type.
```

Comment:
First cut. Soliciting feedback to make sure:
* Developers are not confused when they read this
* Do not have unnecessary POP failures if someone chooses to set this

Reply:
Added more detailed text and warning.